### PR TITLE
Dockerfile: add label-schema and OCI labels

### PR DESCRIPTION
**Pull request by bertsky**
perhaps better this way. But I have no idea how `org.opencontainers.image.base.digest` is supposed to be set within Dockerfile (short of another `BUILD_ARG`), so I just left it out.

### CODE REVIEW

**Review by markusweigelt — APPROVED** [review]

**Comment by bertsky** [comment]
Before you merge, can we trigger GHA here and take a look at the labels, comparing before/after?

File: Dockerfile [hunk 1]
```diff
@@ -16,7 +16,9 @@ LABEL org.label-schema.vcs-ref=$VCS_REF
 LABEL org.label-schema.build-date=$BUILD_DATE
 LABEL org.opencontainers.image.vendor="Saxon State and University Library Dresden"
 LABEL org.opencontainers.image.title="OCR-D Controller"
+LABEL org.opencontainers.image.description="Path to network implementation of OCR-D"
 LABEL org.opencontainers.image.source="https://github.com/slub/ocrd_controller"
+LABEL org.opencontainers.image.documentation="https://github.com/slub/ocrd_controller/blob/${VCS_REF}/README.md"
```
Comment (bertsky) [inline Dockerfile:21]: Yes. For ocrd_kitodo, we can reference the mkdocs site at this point.

**Comment by markusweigelt** [comment]
> Before you merge, can we trigger GHA here and take a look at the labels, comparing before/after?

Yes I have created an image to test labels.

https://github.com/slub/ocrd_controller/pkgs/container/ocrd_controller/90000776?tag=labeltest

File: Dockerfile (hunk 1 above)
Comment (markusweigelt) [reply Dockerfile:21]: Unfortunately this documentation is unspecific for the respective module and would leave it here rather with the Readme.

Comment (bertsky) [reply Dockerfile:21]: Ok, never mind.

**Comment by bertsky** [comment]
> Yes I have created an image to test labels.
> 
> https://github.com/slub/ocrd_controller/pkgs/container/ocrd_controller/90000776?tag=labeltest

That's odd. GH does not show any. docker image inspect does show all the OCI labels under `Config`, but there's also a `null` under `ContainerConfig` – no idea what relevance that has.

**Comment by markusweigelt** [comment]
> That's odd. GH does not show any. docker image inspect does show all the OCI labels under `Config`, but there's also a `null` under `ContainerConfig` – no idea what relevance that has.

My internet is so lame i am still downloading :/

**Review by markusweigelt — APPROVED** [review]

**Pull request merged**